Makefile

```diff
@@ -34,6 +34,10 @@ cypress-test: js-build
 	npm run cypress:test
 .PHONY: cypress-test
 
+js-test: $(JS_SENTINAL)
```

Comment:
Noting a duplication here - there is a `jstest` target in `js.mk` that does an `npm test`. Would that common target work? Or does it need to be `npm run test`?

Reply:
Whoops! Yes that's a duplicate. Removed.